application/browser/application_service.cc

```diff
@@ -215,6 +215,16 @@ Application* ApplicationService::GetApplicationByID(
   return NULL;
 }
 
+bool ApplicationService::KillApplicationByID(const std::string& app_id) {
```

Comment:
don't think it's needed: without shared process model it's enough just to quit/kill browser process.